### ISSUE TITLE
Error:  "the following installed NAV extensions are missing" when mounting tenant

### ISSUE THREAD

**Issue by krregg** [MicrosoftDocs/dynamics365smb-devitpro-pb#1546]
I am having a hard time mounting new tenants. For example, I have a tenant called `dev` and for some reason, I would like to have a copy of this tenant. I make a backup of DB for this tenant and then restore it with a different DB name. After that I try to run this command:

```
Mount-NAVTenantDatabase -ServerInstance $serverInstance -Id dev2 -DatabaseName $newDb -DatabaseServer $dbServer
Mount-NAVTenant -ServerInstance $serverInstance -DatabaseName $newDb -DatabaseServer $dbServer -Tenant dev2 -AllowAppDatabaseWrite -OverwriteTenantIdInDatabase
```

Which gives me this error:
```
Mount-NAVTenant : The tenant dev2 could not be mounted because the following installed NAV extensions are missing, matching content could not be found, or have a higher version than those that are pub
lished and synchronized
```

Yes, on main tenant `dev` I already have some apps installed and I would like to keep them on the new tenant as well. The only way I can do this in that way is that I uninstall these apps from tenant `dev`. After I perform the backup, restore, Mount-NAVTenantDatabase, and finally Mount-NAVTenant. This is the only way I can get the new tenant `dev2` up and running. 

In the end, I have to reinstall apps back on tenant `dev`, but on tenant `dev2` there is none of these apps, only System Application, Base Application, and Application. Any other apps I have to publish again.

I was reading documentation about tenants and somehow I can't get it to work as I want. In general, I would like to make a copy of any tenant and mount it back on application DB without affecting the main tenant. Is there something I am missing here? I would appreciate any help.

I am working with BC 17.0 OnPrem, version W1.

Thank you in advance.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a82ea2b7-69ac-766c-7f27-5cc4ce9f5666
* Version Independent ID: c80b9a5e-7356-6432-4016-f250536c9d99
* Content: [Mount-NAVTenant (Microsoft.Dynamics.Nav.Management) - Dynamics NAV](https://docs.microsoft.com/en-us/powershell/module/microsoft.dynamics.nav.management/mount-navtenant?view=businesscentral-ps-16)
* Content Source: [businesscentral-ps-16/Microsoft.Dynamics.Nav.Management/Mount-NAVTenant.md](https://github.com/MicrosoftDocs/nav-docs-powershell/blob/live/businesscentral-ps-16/Microsoft.Dynamics.Nav.Management/Mount-NAVTenant.md)
* GitHub Login: @jswymer

**Comment by jswymer** [comment]
Hello, Thanks for the feedback.

I tried this on a 17.0 and 17.2 W1 build and got it to work. The only thing I might have done differently is that I didn't use the Mount-NAVTenantDatabase (which is only used with shared schema for BC online). I only used Mount-NAVTenant. Alternatively, I also did the same using the Export-NAVData/Import-NAVData cmdlets. This approach required that synced all extensions after mounting the new tenant, before importing the data.
